client/settings/express-checkout-settings/test/payment-request-button-preview.test.js

```diff
@@ -39,6 +39,19 @@ jest.mock( 'wcpay/data', () => {
 	};
 } );
 
+const render = ( ui, options ) =>
+	baseRender( ui, {
+		wrapper: ( { children } ) => (
+			<div
+				id="wcpay-express-checkout-settings-container"
```

Comment:
This element is rendered here ( https://github.com/Automattic/woocommerce-payments/blob/a920abdca8624cd96174c1aeba277835e6cdd564/includes/class-wc-payment-gateway-wcpay.php#L883-L883 ), and I needed to add this wrapper to more closely match the behavior of the components when in a non-test environment.